app/views/articles/show.html.erb

```diff
@@ -1,17 +1,64 @@
-<h1><%= @article.title %></h1>
+<div class="d-flex justify-content-between align-items-center mt-4">
+  <h1><%= @article.title %></h1>
+  <div class="dropdown show">
+    <a class="btn btn-primary dropdown-toggle"
+       href="#"
+       role="button"
+       id="dropdownMenuLink"
+       data-bs-toggle="dropdown"
+       aria-haspopup="true"
+       aria-expanded="false">
+      <%= t('actions.self') %>
+    </a>
+    <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+      <div class="dropdown-item">
+        <%= link_to model_action('article', 'edit'),
+                    edit_article_path(@article),
+                    class: "link-success link-underline link-underline-opacity-0",
+                    data: { turbo_frame: 'modal' } %>
+      </div>
+      <div class="dropdown-item">
+        <%= link_to model_action('article', 'delete'), article_path(@article), data: {
+          turbo_method: :delete,
+          turbo_confirm: t('form_confirmation')
+        }, class: "link-danger link-underline link-underline-opacity-0" %>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="vh-40">
+  <p><%= @article.body %></p>
+</div>
 
-<p><%= @article.body %></p>
-
-<ul>
-  <li><%= link_to t('actions.edit'), edit_article_path(@article) %></li>
-  <li><%= link_to t('actions.delete'), article_path(@article), data: {
-    turbo_method: :delete,
-    turbo_confirm: t('form_confirmation')
-  } %></li>
-</ul>
-
-<h2><%= t('comment.comments') %></h2>
-<%= render @article.comments %>
-
-<h2><%= t('comment.comments') %></h2>
-<%= render 'comments/form' %>
+<div class="container">
+  <div class="row d-flex justify-content-center">
+    <div class="col-md-12 col-lg-10 col-xl-8">
+      <h2><%= t('comment.comments') %></h2>
+    </div>
+  </div>
+</div>
+<section>
+  <div class="container py-3">
+    <div class="row d-flex justify-content-center">
+      <div class="col-md-12 col-lg-10 col-xl-8">
+        <div class="card">
+          <%= render @article.comments %>
```

Comment:
You need to send here only not archived comments.
You can create a `scope` in `Comment` model for ex, `:active_comments` to filter not archived comments. And you can use it here like `@article.comments.active_comments`.
Read how to write scopes in models.